### DISCUSSION TITLE
Update primary button colour to dark blue

### DISCUSSION THREAD

**Discussion by rnickr** [nhsevidence/NICE-Experience#290] [Ideas]
We're proposing to change the primary button colour from black to the brand dark blue (#00436c).

**Why?**

We currently have 4 button styles: 

CTA - Yellow
Primary - Black
Secondary- Grey
Inverse - White (for use on dark backgrounds)

See [https://design-system.nice.org.uk/storybook/?path=/story/foundations-button--cta](https://design-system.nice.org.uk/storybook/?path=/story/foundations-button--cta )

The Comms team have been asked to implement blue buttons that match brand colours as part of marketing pages for marketing campaigns. They have used inline styles to do this which we want to avoid if possible: https://www.nice.org.uk/plan-prioritise-monitor

![image](https://user-images.githubusercontent.com/56125528/198080204-25655245-8c6f-4416-b1f2-89956f280fce.png)

We could: 

- Insist on only using the existing button colours (inflexible and doesn't support brand work)
- Create another button variant (we should try and avoid creating new variants unless absolutely necessary.)
- Change the primary button colour from black to blue. 

**What impact will this have**

The purpose of the different button styles is to create a visual hierarchy that supports different levels of action on the page. 
The CTA button supports key actions that need to stand out the most. The primary button should have less prominence than the CTA button but more prominence than the secondary button.  

The proposed new button colour: #00436c and existing primary button colour #404040 have similar luminosity values, so the change should keep the hierarchy between the different button styles in tact.  

Making the Primary button blue #00436c also creates more of a distinction between the primary and secondary buttons compared to the current black (#404040) and grey (#666666) colours.

**Comment by ediblecode** [comment]
Thanks for putting this together @rnickr. Great rationale. I think the blue makes a lot of sense to distinguish primary from secondary a bit more whilst aligning with the brand.

**Comment by DesignMagnetic** [comment]
I prefer the blue, the yellow had knock on effects for comment collection, where it made the comments section less distinctive, because of the new yellow. I know we shouldn't always create new variants, but in this instance, I think it would help with a few issues.

**Reply by rnickr** [reply]
@DesignMagnetic  - The plan is to keep the CTA button as yellow and swap the current Primary button (black) to dark blue, so no new variant and still using Yellow for CTA button

**Reply by rnickr** [reply]
@DesignMagnetic Be great if the comment collection elements were documented and added to the Design System so there's visibility. Would help with discussions around issues like this. Also no reason the CTA button on the comment collection couldn't be a primary button instead. 

![image](https://user-images.githubusercontent.com/56125528/199950735-1365789c-94ea-4427-90f8-c266a933452c.png)

**Comment by clarissagardner-NICE** [comment]
I think the rationale for this is laid out really clearly so I see no reason why we shouldn't change the colour, creating another variant seems counterproductive
